src/components/players/_shared/card.js

```diff
@@ -4,7 +4,7 @@ import placeholderImg from '../../../assets/imgs/placeholder.png';
 
 import styles from './card.module.scss';
 
-const Card = ({ player }) => {
+const Card = ({ player, playerTeam }) => {
```

Comment:
Technically this should be called `playerTeamId` as that is what you're sending.